README.md

```diff
@@ -1070,11 +1076,13 @@ This repository is being kept up to date thanks to the help from the community.
 🌻 [Nikola Telkedzhiev](https://github.com/ntelkedzhiev),
 🌻 [Vitor Godoy](https://github.com/vitordagamagodoy),
 🌻 [Manish Saraan](https://github.com/manishsaraan),
-🌻 [Sangbeom Han](https://github.com/uronly14me)
+🌻 [Sangbeom Han](https://github.com/uronly14me),
+🌻 [blackmatch](https://github.com/blackmatch)
 
 ### Stars <br/>
 
-⭐ [Kyle Martin](https://github.com/js-kyle)
-⭐ [Keith Holliday](https://github.com/TheHollidayInn)
+⭐ [Kyle Martin](https://github.com/js-kyle),
+⭐ [Keith Holliday](https://github.com/TheHollidayInn),
+⭐ [Corey Cleary](https://github.com/coreyc)
```

Comment:
Similarly, coreyc's first PR was merged in November, although some commits were authored in October.